src/cart/crt_iv.c

```diff
@@ -1879,6 +2026,15 @@ crt_hdlr_iv_sync(crt_rpc_t *rpc_req)
 		D_GOTO(exit, rc = -DER_NONEXIST);
 	}
 
+	/* Check group version match */
+	grp_ver = ivns_internal->cii_grp_priv->gp_membs_ver;
+	if (grp_ver != input->ivs_grp_ver) {
+		D_ERROR("Group (%s) version mismatch. Local: %d Remote :%d\n",
+			ivns_id.ii_group_name, grp_ver,
+			input->ivs_grp_ver);
+		D_GOTO(exit, rc = -DER_GRPVER);
+	}
```

Comment:
Same here, i dont think this check is needed as we are not computing parent/children anywhere in this call

Reply:
I switched comments around with previous.  But once again. it is comparing the group version number associated with an incoming rpc request with the node's current version.  If this type of check is not valid, then I will eliminate.